nest-cli/src/main.rs

```diff
@@ -57,6 +57,82 @@ fn main() {
                         .required(true),
                 )
         )
+        .subcommand(
+            SubCommand::with_name("group")
+                .setting(AppSettings::SubcommandRequiredElseHelp)
+                .about("Operate on groups")
+                .subcommand(
+                    SubCommand::with_name("add")
+                        .about("Create new groups")
+                        .arg(
+                            Arg::with_name("GROUP")
+                                .help("Groups to create")
+                                .multiple(true)
+                                .required(true),
+                        )
+                        .arg(
+                            Arg::with_name("PARENT")
```

Comment:
I think it's more intuitive to use [Arg::default_value()](https://docs.rs/clap/2.32.0/clap/struct.Arg.html#method.default_value) instead of `Option::unwrap_or()` like you did a bit later.